lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+require "csv"
+
+class Menu
+
+  attr_reader :menu
```

Comment:
This class is the menu, so it shouldn't have an attribute called `menu` as that leads us to `menu.menu`.  This class should provide methods to query the menu data, not just expose the array.